### PR TITLE
feat: Initial generation of google-cloud-monitoring-metrics_scope-v1

### PR DESCRIPTION
**Pull request by dazuma**
Auto-created at 2021-09-27 15:41:14 +0000 using the toys pull request generator.

### CODE REVIEW

File: google-cloud-monitoring-metrics_scope-v1/.repo-metadata.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "client_documentation": "https://googleapis.dev/ruby/google-cloud-monitoring-metrics_scope-v1/latest",
+    "distribution_name": "google-cloud-monitoring-metrics_scope-v1",
+    "language": "ruby",
+    "name": "monitoring",
+    "name_pretty": "Cloud Monitoring Metrics Scopes V1 API",
+    "product_documentation": "https://cloud.google.com/monitoring",
+    "repo": "googleapis/google-cloud-ruby",
+    "requires_billing": true,
+    "ruby-cloud-description": "Cloud Monitoring collects metrics, events, and metadata from Google Cloud, Amazon Web Services (AWS), hosted uptime probes, and application instrumentation. The Metrics Scopes API manages the list of monitored projects and accounts. Note that google-cloud-monitoring-metrics_scope-v1 is a version-specific client library. For most uses, we recommend installing the main client library google-cloud-monitoring instead. See the readme for more details.",
+    "ruby-cloud-product-url": "https://cloud.google.com/monitoring",
```
Comment (quartzmo) [inline google-cloud-monitoring-metrics_scope-v1/.repo-metadata.json:12]: Possibly a better landing page: https://cloud.google.com/monitoring/settings/manage-api

**Review by quartzmo — APPROVED** [review]
Approved, with inline suggestion for product docs url.

Also, I think the library and module should have been `metrics_scopes` instead of `metrics_scope`, but I assume it's too late to change that.

File: google-cloud-monitoring-metrics_scope-v1/.repo-metadata.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "monitoring.googleapis.com",
+    "client_documentation": "https://googleapis.dev/ruby/google-cloud-monitoring-metrics_scope-v1/latest",
+    "distribution_name": "google-cloud-monitoring-metrics_scope-v1",
+    "language": "ruby",
+    "name": "monitoring",
+    "name_pretty": "Cloud Monitoring Metrics Scopes V1 API",
+    "product_documentation": "https://cloud.google.com/monitoring",
```
[the remark this answers is not in the document]
Comment (dazuma) [reply google-cloud-monitoring-metrics_scope-v1/.repo-metadata.json:8]: Good find. I'll get that updated upstream.

**Comment by dazuma** [comment]
> Also, I think the library and module should have been `metrics_scopes` instead of `metrics_scope`, but I assume it's too late to change that.

Yeah, unfortunately we can't change it at this point without disruption.

**Pull request merged**